superset/assets/src/dashboard/components/gridComponents/Tabs.jsx

```diff
@@ -100,7 +101,14 @@ class Tabs extends React.PureComponent {
     }
   }
 
-  handleClickTab(tabIndex) {
+  handleClickTab(tabIndex, ev) {
+    const target = ev.target;
+    // special handler for clicking on anchor link (or whitespace nearby):
+    // only show URLShortLinkButton but do not change tab
+    if (ANCHOR_LINK_TAGNAMES.includes(target.tagName)) {
```

Comment:
instead of targeting the `tagName`, could we use something more specific like a CSS class or id?

Reply:
I added a CSS class. also added a unit test.